server/.eslintrc.js

```diff
@@ -0,0 +1,61 @@
+const tabIndentation = ['error', 'tab'];
+
+module.exports = {
+	root: true,
+	env: {
+		node: true,
+		jest: true,
+	},
+	parser: "@typescript-eslint/parser",
+	settings: {
+		// https://github.com/alexgorbatchev/eslint-import-resolver-typescript
+		'import/parsers': {
+			'@typescript-eslint/parser': ['.js', '.ts']
+		},
+		'import/resolver': {
+			// use <root>/tsconfig.json
+			typescript: {},
+		}
+	},
+
+	plugins: [
+		'@typescript-eslint',
+		'eslint-comments',
+		'jest',
+		'promise',
+		'unicorn',
+		'import',
+	],
+
+	extends: [
+		'airbnb-typescript/base',
```

Comment:
Is het nodig dat we zowel de airbnb base als de recommended extenden?
Zou verwachten dat die airbnb base min of meer hetzelfde doet, niet?